ceph-dev-pipeline/build/Jenkinsfile

```diff
@@ -2,15 +2,41 @@ ceph_build_repo = "https://github.com/ceph/ceph-build"
 ceph_build_branch = "main"
 base_node_label = "gigantic"
 ubuntu_releases = [
-  "noble", // 24.04
-  "jammy", // 22.04
-  "focal", // 20.04
+  "noble": "24.04",
+  "jammy": "22.04",
+  "focal": "20.04",
 ]
```

Comment:
Square brackets for map hurts my json/python brain, but I guess it's right for this weird language :-)